### PR TITLE
adds diagnostic hud sunglasses 

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
🆑 :
add: you can add diagnostic hud to sunglasses
sprite: fixes medhud sunglasses having cyan squares instead of blue when worn
/:cl:

FAQ
q)does this make sunglasses craftable? 
r)no

q)does this nerf cyborgs?
r) no

q)are the sprites subject to copyrights?
r) no

q) was this made to powergame?
r) no i just wanted cool looking orange sunglasses

q) is bikeshredding prohibited in this pr?
r) yes

### CODE REVIEW

**Comment by SgtxRootbeer** [comment]
I promise I will not bikeshred

**Comment by k-k-karma** [comment]
supreme web editor

**Comment by SgtxRootbeer** [comment]
maxresdefault 5

**Comment by plapatin** [comment]
dont fix medhud glasses i want to use my balaclava and medhud glasses and tell people im sans undertale more

**Comment by ShizCalev** [comment]
Conflict.

**Comment by Anonmare** [comment]
Looks fine to me

**Comment by Tlaltecuhtli** [comment]
@ShizCalev i tried swapping clean dmi from the master and then applying the extra icons, but it still gives conflicts

**Comment by yorii** [comment]
do it again, someone probably merged something while you did that

**Comment by SpaceManiac** [comment]
you can't just take the clean dmi from master, your whole branch needs to be up to date

**Comment by SpaceManiac** [comment]
Closing as stale/conflict, contact a maintainer if you want it reopened

**Comment by subject217** [comment]
i'll fix this and PR it if tlal doesn't want to do it

File: code/modules/crafting/recipes.dm [hunk 1]
```diff
@@ -660,6 +660,24 @@
...
+	name = "Diagnostic HUDsunglasses"
+	result = /obj/item/clothing/glasses/hud/diagnostic/sunglasses
+	time = 20
+	tools = list(TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
+	reqs = list(/obj/item/clothing/glasses/hud/diagnostic = 1,
+				  /obj/item/clothing/glasses/sunglasses = 1,
+				  /obj/item/stack/cable_coil = 5)
+	category = CAT_CLOTHING
+
+/datum/crafting_recipe/hudsunmedremoval
```
Comment (subject217) [inline code/modules/crafting/recipes.dm]: needs to be renamed, you copy/pasted the medical one

**Comment by subject217** [comment]
https://github.com/tgstation/tgstation/pull/41491

**Pull request closed without merging**